Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js

```diff
@@ -2073,8 +2074,9 @@ Ext.onReady(function () {
 
     // if there is no history, select the top node
     if (!Ext.History.getToken()) {
-        var node = Ext.getCmp('devices').getRootNode();
-        node.fireEvent('expand', node);
```

Comment:
select root node instead of fire "expand" event, it wasn't working before and didn't select anything;